apps/frontend/src/Screens/MainScreen.tsx

```diff
@@ -55,7 +55,7 @@ const MainScreen = () => {
   }, []);
   return(
   <>
-  <Navbar/>
+  {/*<Navbar/>*/}
```

Comment:
Probably we can just remove this line.

Reply:
Sorry, I meant to delete that. It's fixed now.